angular-notify.css

```diff
@@ -23,6 +22,18 @@
     box-shadow: 0 6px 12px rgba(0,0,0,.175);
 }
 
+.cg-notify-message-center {
+    left:50%;
+}
+
+.cg-notify-message-left {
+    left:1%;
```

Comment:
Can both this and the right class use a px left and right.  10 or15px most likely?